lti_consumer/lti_consumer.py

```diff
@@ -849,11 +849,11 @@ def _get_context_for_template(self):
             'has_score': self.has_score,
             'weight': self.weight,
             'module_score': self.module_score,
-            'comment': self.ugettext(sanitized_comment),
+            'comment': sanitized_comment,
```

Comment:
@douglashall yeah i think we should remove it although it has default values but still it is coming from database. Removed.